rpcclient/wallet.go

```diff
@@ -312,6 +313,30 @@ func (c *Client) ListSinceBlockMinConf(blockHash *chainhash.Hash, minConfirms in
 	return c.ListSinceBlockMinConfAsync(blockHash, minConfirms).Receive()
 }
 
+// ListSinceBlockMinConfWatchOnlyAsync returns an instance of a type that can be used to
+// get the result of the RPC at some future time by invoking the Receive
+// function on the returned instance.
+//
+// See ListSinceBlockMinConfWatchOnly for the blocking version and more details.
+func (c *Client) ListSinceBlockMinConfWatchOnlyAsync(blockHash *chainhash.Hash, minConfirms int, watchOnly bool) FutureListSinceBlockResult {
+	var hash *string
+	if blockHash != nil {
+		hash = btcjson.String(blockHash.String())
+	}
```

Comment:
This probably won't work if you want to do something like _list all watch-only transactions_ with the following call:
```
ListSinceBlockMinConfWatchOnlyAsync(nil, 1, true)
```

Here's a workaround, although I'm not sure if it's the best solution.

```suggestion
	} else {
	        hash = btcjson.String("")
	}
```

Reply:
Thanks to https://github.com/btcsuite/btcd/pull/1594, we may not need to do this. Could you please verify this on your side?